packages/workspace/src/core/project-graph/project-graph.ts

```diff
@@ -188,7 +188,7 @@ export function createProjectGraph(
   const cacheEnabled = process.env.NX_CACHE_PROJECT_GRAPH !== 'false';
   let cache = cacheEnabled ? readCache() : false;
   assertWorkspaceValidity(workspaceJson, nxJson);
-  const normalizedNxJson = normalizeNxJson(nxJson);
+  const normalizedNxJson = normalizeNxJson(nxJson, workspaceJson);
```

Comment:
Are we passing workspace.json to extract implicit deps? Is it because otherwise it would have been a breaking change?

Reply:
I'm not sure if this is the case. createContext merges nx.json in, but I imagine project configs should be combined there, and shouldRecomputeWholeGraph can be updated.

Reply:
We aren't extracting implicit deps here, rather the normalizeNxJson requires a list of the projects to expand wildcards in the nx.json implicitDeps object (not project specific ones).

This is most likely a bug in the current version that has went unreported, with wildcard implicit dependencies missing projects that are not present in nx.json